### ISSUE TITLE
feature: add "perlmutter" tag after counting

### ISSUE THREAD

**Issue by ercius** [OpenChemistry/distiller#1196]
Streaming sessions show a perlmutter tag with the file path once the file is available on CFS:
![image](https://github.com/user-attachments/assets/d10196d7-7da3-4014-a629-6770918850df)

Can we add the same tag after a regular counting job is finished.

**Comment by cjh1** [comment]
Yes, I think this would be quite a simple change. We already have the logic todo this for transfer jobs, we can just do it for counting jobs as well.

**Comment by ercius** [comment]
This would make a big improvement to the workflow and user experience.

**Comment by cjh1** [comment]
Fixes in: #1198

**Comment by ercius** [comment]
The perlmutter tag is being shown, but the location is incorrect. Here is the location for a data set that was counted normalled:
![image](https://github.com/user-attachments/assets/b6c2af9d-77a6-464f-a175-90ef3fb80206)

There is no file name and the folder is `raw` instead of `counted`. The date is right!

**Comment by cjh1** [comment]
OK, I will look at fixing this.

**Comment by cjh1** [comment]
This is resolve by #1217

**Comment by ercius** [comment]
confirmed that this is working. Very nice addition!
